src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -11,11 +11,13 @@
          This is probably because the code coverage target invocation does not canonicalize the
          process name. -->
     <ToolHost Condition="'$(OS)' == 'Windows_NT'">CoreRun.exe</ToolHost>
-    <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
```

Comment:
It should be all lower case. This was fixed recently in our packages to ensure it is all lowercase 'corerun'.